src/jit/importer.cpp

```diff
@@ -3714,6 +3714,34 @@ GenTreePtr Compiler::impIntrinsic(GenTreePtr            newobjThis,
             break;
         }
 
+        case CORINFO_INTRINSIC_GetRawHandle:
+        {
+            assert(IsTargetAbi(CORINFO_CORERT_ABI)); // Only CoreRT supports it.
+            CORINFO_RESOLVED_TOKEN resolvedToken;
+            resolvedToken.hMethod      = method;
```

Comment:
You can null it out. We can fetch that from token/tokenScope (and in fact, we need do it from the token anyway if the method is canonical (and it pretty much always will be canonical)).